demo/helpers.ts

```diff
@@ -17,7 +17,7 @@ type GridResponse = {
         metadata: {
           description: string;
           suppliersReference: string;
-          source: string;
+          credit: string;
```

Comment:
just for my understanding, what's the reason for renaming here?

Reply:
I think this was broken before. It may be that the grid's api changed underneath

Reply:
Oh, oh, oh. This hits at our Grave Confusion. Basically, Grid’s `credit` is what CAPI (and maybe FlexAPI?) calls `source`. Can’t comment on code, but can bore to death with details…